### DISCUSSION TITLE
Manually adjust the position of the Handle?

### DISCUSSION THREAD

**Discussion by Holybasil** [xyflow/xyflow#1790] [New Features]
🤔

**Comment by cliffordfajardo** [comment]
Hi @Holybasil ,
Would you be able to share more details & an example scenario?

I have an old code sandbox example showing different placements of the handle on a node here
- https://codesandbox.io/s/staging-fire-wcekf?file=/src/examples/Example1/index.tsx

<img width="1125" alt="CleanShot 2021-12-25 at 13 26 32@2x" src="https://user-images.githubusercontent.com/6743796/147393850-9fdcbbfa-5911-40eb-99dd-c3644e9e4d90.png">


If this isn't what you're looking for I also recommend you the codesandbox `react-flow-renderer` examples that's public, there might be more examples there
- https://codesandbox.io/examples/package/react-flow-renderer

<img width="1277" alt="CleanShot 2021-12-25 at 13 39 50@2x" src="https://user-images.githubusercontent.com/6743796/147394003-d022ed6e-e210-494a-bacd-e925ac238b0a.png">

**Comment by Holybasil** [comment]
Actually, what I’m trying to say is that if I drag and drop to make the vertical nodes parallel, can I also drag and drop to change the location of the **Handle** to make the **Edge** look better.

Not
<img width="499" alt="CleanShot 2021-12-27 at 10 22 40@2x" src="https://user-images.githubusercontent.com/21138718/147429379-4abbcd8e-0a7e-4843-b856-ea0de69c2ba4.png">
But
<img width="514" alt="CleanShot 2021-12-27 at 10 49 25@2x" src="https://user-images.githubusercontent.com/21138718/147429399-b82a17d1-c98a-4e7a-a14c-360650bf6fcd.png">

That’s what I got by changing the code. Can I do that by dragging and dropping?

https://codesandbox.io/s/mini-react-flow-renderer-forked-uy2cv
